css/pages/index.css

```diff
@@ -4,6 +4,10 @@
   box-sizing: border-box;
 }
 
+.index-body {
```

Comment:
We can directly use the element selector here.
```
body {
  
}
```

Reply:
Done